INSTALL.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# 
+# 
+# Installation script for PyKaldi
+# 
+# Usage:
+# 	./INSTALL.sh [python] [pip] [python_library]
+# 
+# 	python, pip (optional) point to the location of the executables. Defaults to $(which ..)
+# 	python_library (optional) location of the libpython shared library. As default runs a script to find it.
+# 
+# 
+
+set -e -x 
+INSTALL_NINJA=true
+
+#######################################################################################################
+# Installation configuration
+# This determine where things are going to get installed
+#######################################################################################################
```

Comment:
We need a check here making sure that this script is being called inside the pykaldi directory. If not error out.